examples/07-EMIT/ComputeChannelSeparation.py

```diff
@@ -0,0 +1,443 @@
+installed_packages = [r.decode().split('==')[0] for r in reqs.split()]
+
+# Install required packages if they are not installed
+def install(package): 
+    subprocess.check_call([sys.executable, '-m', 'pip', 'install', package])
+
+# Install any missing libraries
+required_packages = ['plotly', 'tqdm', 'matplotlib', 'numpy']
+for package in required_packages:
+    if package not in installed_packages:
+        install(package)
+
+# Import required modules
+import plotly.graph_objects as go
+from tqdm.notebook import tqdm
+    
+from matplotlib import pyplot as plt
+plt.ion() # enables interactive mode so plots show immediately
+plt.show()
+import numpy as np
+
+###############################################################################
+# Set non-graphical mode
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Set non-graphical mode. ``"PYAEDT_NON_GRAPHICAL"``` is needed to generate
+# documentation only.
+# You can set ``non_graphical`` either to ``True`` or ``False``.
+# The ``NewThread`` Boolean variable defines whether to create a new instance
+# of AEDT or try to connect to existing instance of it if one is available.
+
+non_graphical = os.getenv("PYAEDT_NON_GRAPHICAL", "False").lower() in ("true", "1", "t")
+NewThread = False
+desktop_version = "2023.2"
+
+###############################################################################
+# Launch AEDT with EMIT
+# ~~~~~~~~~~~~~~~~~~~~~
+# Launch AEDT with EMIT. The ``Desktop`` class initializes AEDT and starts it
+# on the specified version and in the specified graphical mode.
+
+d = pyaedt.launch_desktop(desktop_version, non_graphical, NewThread)
+emitapp = Emit(pyaedt.generate_unique_project_name())
+
+###############################################################################
+# Create and connect EMIT components
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Set up the scenario with radios connected to antennas
+
+def add_and_connect_radio(radio_name, schematic_name=""):
+    """Add a radio from the EMIT library and connect
+    it to an antenna.
+    Returns: 
+        Instance of the radio.
+    Argments:
+        radio_name - String name of the EMIT library radio
+            to add.
+        schematic_name - Name that appears in the schematic.
+    """
+    rad = emitapp.modeler.components.create_component(radio_name, schematic_name)
+    ant = emitapp.modeler.components.create_component("Antenna")
+    if rad and ant:
+        ant.move_and_connect_to(rad)
+    return rad
+
+# Add three systems to the project
+bluetooth = add_and_connect_radio("Bluetooth Low Energy (LE)", "Bluetooth")
+gps = add_and_connect_radio("GPS Receiver", "GPS")
+wifi = add_and_connect_radio("WiFi - 802.11-2012", "WiFi")
+
+###############################################################################
+# Configure the radios
+# ~~~~~~~~~~~~~~~~~~~~
+# Enable the HR-DSSS bands for the wifi radio and set the power level
+# for all transmit bands to simulate coupling.
+def set_band_power_level(band, power):
```

Comment:
@jsalant22 Does it make sense to move these two functions into the API, alongside band.enabled? That would allow others to use it and would simplify this example.

Reply:
@myoung301 I'm working on the pyAedt units, can I move these functions on that branch since I'll need to update the functions again anyway? 

I'll likely have other example updates with those changes too.

Reply:
@jsalant22 Yeah that sounds good to me.